jablib/src/main/java/org/jabref/logic/search/indexing/BibFieldsIndexer.java

```diff
@@ -178,9 +178,8 @@ public void addToIndex(Collection<BibEntry> entries, BackgroundTask<?> task) {
                 return;
             }
             addToIndex(entry);
-            task.updateProgress(i, entries.size());
+            task.updateProgress(i++, entries.size());
```

Comment:
From a programmer perspective this is right and the message is still displaying the right number.
But from the perspective of a debugger this troubles me. Two seperate iterators in the same loop.
We have the for each in l 175 and int i. I think maybe a traditional for(int i; ...) might be better, so we only have on source of truth for the interator in the loop?